src/osd/OSD.cc

```diff
@@ -8261,7 +8261,15 @@ void OSD::handle_osd_map(MOSDMap *m)
           << "," << m->newest_map << "]"
 	  << dendl;
 
-  logger->inc(l_osd_map);
+  logger->inc(l_osd_map); 
+  if (!m->maps.empty()) {
+    logger->inc(l_osd_full_map_received, m->maps.size()); 
+  }
+  if (!m->incremental_maps.empty()) {
+    logger->inc(l_osd_inc_map_received, m->incremental_maps.size());
+  } 
+  dout(10) << __func__ << ": received " << m->maps.size() << " full maps and " << m->incremental_maps.size() << " incremental maps " << dendl;
```

Comment:
Ah, sorry, one more nit: limit lines to 80 characters.  See the debug lines immediately above for a common way of breaking up debug lines.